tests/zfs-tests/tests/perf/perf.shlib

```diff
@@ -122,33 +122,33 @@ function do_fio_run_impl
 	# disable client cache for reads.
 	if [[ $NFS -eq 1 ]]; then
 		export DIRECT=1
-		do_setup_nfs $script
+		do_setup_nfs "$script"
 	else
 		export DIRECT=0
 	fi
 
 	# This will be part of the output filename.
-	typeset suffix=$(get_suffix $threads $sync $iosize)
+	typeset suffix=$(get_suffix "$threads" "$sync" "$iosize")
 
 	# Start the data collection
-	do_collect_scripts $suffix
+	do_collect_scripts "$suffix"
 
 	# Define output file
 	typeset logbase="$(get_perf_output_dir)/$(basename \
-	    $SUDO_COMMAND)"
+	    "$SUDO_COMMAND")"
 	typeset outfile="$logbase.fio.$suffix"
 
 	# Start the load
 	if [[ $NFS -eq 1 ]]; then
-		log_must ssh -t $NFS_USER@$NFS_CLIENT "
+		log_must ssh -t "$NFS_USER"@"$NFS_CLIENT" "
 			fio --output-format=${PERF_FIO_FORMAT} \
 			    --output /tmp/fio.out /tmp/test.fio
 		"
-		log_must scp $NFS_USER@$NFS_CLIENT:/tmp/fio.out $outfile
-		log_must ssh -t $NFS_USER@$NFS_CLIENT "sudo -S umount $NFS_MOUNT"
+		log_must scp "$NFS_USER"@"$NFS_CLIENT":/tmp/fio.out "$outfile"
+		log_must ssh -t "$NFS_USER"@"$NFS_CLIENT" "sudo -S umount $NFS_MOUNT"
 	else
-		log_must fio --output-format=${PERF_FIO_FORMAT} \
-		    --output $outfile $FIO_SCRIPTS/$script
+		log_must fio --output-format="${PERF_FIO_FORMAT}" \
+		    --output "$outfile" "$FIO_SCRIPTS"/"$script"
```

Comment:
Here and below, wo we need to close/reopen quotes around `/`?

Reply:
This is the same situation as above, "$FIO_SCRIPTS"/"$script" and "$FIO_SCRIPTS/$script" will behave the same way